data/json/recipes/armor/torso.json

```diff
@@ -1764,7 +1765,7 @@
     "skills_required": [ "fabrication", 5 ],
     "time": "10 h 30 m",
     "autolearn": true,
-    "using": [ [ "sewing_standard", 100 ], [ "fabric_standard_nostretch", 14 ] ],
+    "using": [ [ "sewing_standard", 100 ], [ "tailoring_cotton_patchwork", 16 ] ],
```

Comment:
Why did you leave sewing standard here when you adjusted it out in previous recipes?

Reply:
I missed it because i was tired bahah. I've resolved it with the next commit i'm pushing after i lint.